### ISSUE TITLE
gdcl-py - gdcl in Pytnon

### ISSUE THREAD

**Issue by ghost** [dohliam/gdcl#13]
Is it possible rewrite original gdcl to Python?

**Comment by dohliam** [comment]
Absolutely! The capabilities of both languages are quite similar so I don't think there is any reason why it couldn't be done. :smile:

**Comment by ghost** [comment]
So, I has idea on creating cutted version of GoldenDict for using Python3 and PyQt5.

There are few projects that I want use as base:
- Python3
  - http://github.com/prodotiscus/iron-dict/
- Ruby
  - https://github.com/dohliam/gdcl/
- Qt/C++
  - https://github.com/kernel1024/goldendict-lib/

I want make dictionary app limited to DSL dictionary format support.
- Format support:
  - raw DSL - `.dsl`;
    - https://github.com/Tvangeste/SampleDSL/
    - https://github.com/open-dsl-dict/
  - DSL compressed with dictzip/dictizip;
    - https://github.com/bauman/python-idzip
  - some online dictionary (optional);
- Platforms:
  - GUI (PyQt5):
    - Mobile & Tablet UI (Android);
    - Desktop UI (Linux, macOS, etc.);
  - CLI (pure Python3):
    - run on Android using Termux with `python`  (Python3) package installed;
    - run on Desktop with Python3 installed from terminal (bash, cmd, etc.);

Could you give me some advice on what should I do? Maybe, you would be interested in such app development to?
